rfcs/models/security/confidentiality.md

```diff
@@ -25,20 +25,24 @@ As the figure below illustrates, the data sharing protocol first involves a roun
 
 <img src="../../resources/images/e2e-confidentiality.png" width=100%>
```

Comment:
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/17192099/169480164-293f1fdd-4588-41ba-91a2-882d0ccb2c42.png">
This is how the image looks in dark mode, is it good? or do you want to edit it? (I use dark mode always, and thus noticing some texts slightly faint, its okay for me though). Also the backgrounds of hash, peer are not transparent, but white.

Reply:
This applies to all figures throughout this repository. We should submit a separate PR, adding a white background to every image.

Reply:
Opened issue #260 to track.